src/components/Layout/Sidebar/SidebarLink.tsx

```diff
@@ -72,12 +73,16 @@ export function SidebarLink({
         }
       )}>
       {/* This here needs to be refactored ofc */}
-      <span
-        className={cn({
-          'text-gray-400 dark:text-gray-500': wip,
-        })}>
-        {title}
-      </span>
+      <div>
+        {title}{' '}
+        {canary && (
+          <IconCanary
+            title="This feature is available in the latest Canary"
+            className="ml-2 text-gray-30 dark:text-gray-60 inline-block w-4 h-4"
+          />
+        )}
```

Comment:
I feel like we can choose to hide the icon near title as we are showing a callout + icon on the left pane?